docker-compose.yml

```diff
@@ -961,6 +962,14 @@ services:
     image: appwrite/browser:0.3.1
     networks:
       - appwrite
+  
+  appwrite-geo:
+    container_name: appwrite-geo
+    image: appwrite/geo:0.1.0
+    networks:
+      - appwrite
+    environment:
+      - GEO_SECRET=$_APP_GEO_SECRET
```

Comment:
Just double checking,
- Does this container have healthcheck?
- Does this container have security protections - Auth header or JWT?